### DISCUSSION TITLE
Tooltip on touch screen

### DISCUSSION THREAD

**Discussion by alekseykurylev** [adobe/react-spectrum#5613] [Q&A] [answered]
Why shouldn't Tooltip work on touch screen? Are there plans to support touch screen in the future?

**Comment by snowystinger** [comment, marked as answer]
Tooltips are activated by hover and focus. Not press because they're usually on buttons and a press would trigger the buttons action. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role#accessibility_concerns

Why do you ask?

**Reply by alekseykurylev** [reply]
I didn't know that Tooltip has so many limitations. Thanks for your reply.
